packages/containers/src/Action/Action.component.js

```diff
@@ -11,11 +11,13 @@ function Action({ name, ...rest }, context) {
 	const onClick = (event, payload) => {
 		context.store.dispatch(payload.action.payload);
 	};
+	let action;
 	if (name) {
-		const action = api.action.getActionInfo(context, name);
-		return (<PureAction {...action} onClick={onClick} />);
+		action = actions.getProps(context, name, rest.model);
+	} else {
+		action = actions.evalExpressions(rest, context);
 	}
-	return (<PureAction {...rest} onClick={onClick} />);
+	return (<PureAction {...action} onClick={onClick} />);
```

Comment:
To complete my comment on Action component, here we should evaluate expression if present, and if it is falsy, return null.